README.md

```diff
@@ -176,13 +176,17 @@ $ omf install https://github.com/jhillyerd/plugin-git
 | gmt          | `git mergetool --no-prompt`                                 |
 | grh          | `git reset HEAD`                                            |
 | grhh         | `git reset HEAD --hard`                                     |
+| grs          | `git restore`                                               |
+| grss         | `git restore --source`                                               |
```

Comment:
Column is too wide.  Might have been from the use of tabs; if so, please convert to spaces.

Reply:
All done. squash merge if ready